app/apps/handlers.js

```diff
@@ -106,3 +106,17 @@ exports.app_details = [
       .catch(next);
   },
 ];
+
+
+exports.app_delete = [
+  ensureLoggedIn('/login'),
+  (req, res, next) => {
+    const app_id = req.params.app_id;
+
+    api.apps.delete(app_id)
```

Comment:
Don't really need to set a variable here, just do `api.apps.delete(req.params.app_id)`